libs/libc/unistd/lib_chdir.c

```diff
@@ -115,7 +115,7 @@ int chdir(FAR const char *path)
   /* Verify that 'path' refers to a directory */
 
   ret = stat(path, &buf);
-  if (ret != 0)
+  if (ret)
```

Comment:
could you please explain why `!= 0` is wrong?

Reply:
ok，i use coverity scan it.coveriy suggest use ret. it does not mean `!=` is wrong 🐰

Reply:
maybe `ret < 0` then?